### PR TITLE
Missing social icons

### LINKED ISSUE #21490: Missing social icons
Our [PublicizeServiceIcon](https://github.com/wordpress-mobile/WordPress-Android/blob/09d8ee094f97fa3affcc87a7f7ae0995c9b9c6cd/WordPress/src/main/java/org/wordpress/android/ui/publicize/PublicizeServiceIcon.kt#L15) class is missing icons for a few services. We should correct this.

blue

[Related iOS issue](https://github.com/wordpress-mobile/WordPress-iOS/issues/23860)

### PR DESCRIPTION
**Pull request by nbradbury**
Fixes #21490

Our social feature was missing icons for Bluesky, Nextdoor, and Threads. This PR adds the missing icons, and also adds a generic icon for services that don't have an icon in the future.

To test:

* My Site > More > Social
* Verify the icons appear correctly

To view the icons as they appear when enabled, you can change [this line](https://github.com/wordpress-mobile/WordPress-Android/blob/be209f8b43324032924a13da4bcec7ec10093441/WordPress/src/main/java/org/wordpress/android/ui/publicize/adapters/PublicizeServiceAdapter.java#L114) from `> 0` to `== 0`.

light

### CODE REVIEW

**Comment by dangermattic** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">PR is not assigned to a milestone.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_pr-check">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: WordPress/src/main/res/drawable/ic_social_bluesky.xml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="256dp"
```
[the remark this answers is not in the document]
Comment (dcalhoun) [reply WordPress/src/main/res/drawable/ic_social_bluesky.xml]: Should we attempt to downsize this?

**Review by dcalhoun — APPROVED** [review]
Thanks for addressing this. 🙇🏻‍♂️ 

These icons are better than an empty space. However, I can't help but note that all the existing icons are contained within a filled circle. Ideally we update this for consistency at some point.

**Comment by nbradbury** [comment]
> I can't help but note that all the existing icons are contained within a filled circle. Ideally we update this for consistency at some point.

I'm not sure we can. Nextdoor's [brand guidelines](https://about.nextdoor.com/nextdoor-brand-use-guidelines/) state, "Nextdoor trademarks must be used as provided by Nextdoor with no modifications." Instagram is [similar](https://about.meta.com/brand/resources/instagram/instagram-brand/), "Anyone using Instagram’s assets should only use the logos and screenshots found on our Brand Resource Center site and follow these guidelines."

**Comment by dcalhoun** [comment]
> Nextdoor's [brand guidelines](https://about.nextdoor.com/nextdoor-brand-use-guidelines/) state, "Nextdoor trademarks must be used as provided by Nextdoor with no modifications." Instagram is [similar](https://about.meta.com/brand/resources/instagram/instagram-brand/), "Anyone using Instagram’s assets should only use the logos and screenshots found on our Brand Resource Center site and follow these guidelines."

Sure. We could either ignore the guidelines or remove the existing modifications—i.e., the circles surrounding all other logos. I'd be fine with the latter. 

Side note: I'm sure these companies' brand guidelines likely include information discouraging colorizing the logos, which we are doing based on each connection's status.

**Comment by nbradbury** [comment]
> Should we attempt to downsize this?

Good call. I reduced it to 64 x 64.

**Pull request merged**